### PR TITLE
Added covetrus logo to generated app header

### PR DESCRIPTION
**Pull request by dominictracey**
## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
- [ ] Double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

File: internals/scripts/setup.js [hunk 1]
```diff
@@ -1,151 +0,0 @@
-#!/usr/bin/env node
```
Comment (ascheffel) [inline internals/scripts/setup.js:1]: I dont think you should remove this. We will need this later

Comment (ascheffel) [reply internals/scripts/setup.js]: Please leave this, you also left reference to this in the package.json

Comment (ascheffel) [reply internals/scripts/setup.js:1]: If the goal is to change what the default clean command does, then change the name of the current clean command in the package.json and add your new command. This one is actually very useful it cleans out the default project.

**Comment by dominictracey** [comment]
Ah, I had built a site before I checked in. Need to figure out how to test and contribute from a clean install.

**Comment by ascheffel** [comment]
The react boilerplate way is to basically clone the repo and run clean (wipes out all the sample code). I think most people are used to create react app, where run clean clears the node modules dir. Not sure why you would ever want an actual command to do that. If you are doing that, that often its probably a bad thing. I've seen it happen before though, so happy to rename the built in clean to something else and add the rimraf node_modules as clean.

**Comment by Can-Sahin** [comment]
I dont understand what this is. What's the logo and why would I change it ? What is covetrus anyway?

**Comment by dominictracey** [comment]
Sorry - wrong repo! We are trying to define a react app template for our work and looking at your excellent work. Thanks and sorry again!

**Pull request closed without merging**